sdk/core/azure-core/src/main/java/com/azure/core/spatial/package-info.java

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+/**
+ * Package containing GeoJSON classes.
+ */
+package com.azure.core.spatial;
```

Comment:
Don't forget to export through modules-info too

Reply:
Added